--- ./Src/Core/Interfaces/Entities/IAgressorController.cs
+++ ./Src/Core/Interfaces/Entities/IAgressorController.cs
@@ -5,6 +5,7 @@
 {
     string CurrentState { get; }
     Vector2 CurrentDirection { get; }
+    bool IsOnCooldown { get; }
     void Update(float p_delta, bool p_hasTarget, bool p_hasLineOfSight);
     void UpdateChaseDirection(Vector2 p_agressorPosition, Vector2 p_targetPosition);
     void SetDead();
